docs/_documentation/plugins/share.md

```diff
@@ -21,5 +21,5 @@ On iOS, currently only sharing by linked Twitter account is supported. There is
 
 A sample using the Share plugin is:
 
-- Conference - see https://github.com/slodge/MvvmCross-Tutorials/tree/master/Sample%20-%20CirriousConference
+- Conference - see hhttps://github.com/slodge/MvvmCrossConference
```

Comment:
I can't find the original link to the Slodge repository. The example i found contain the a example of use of the Sharing plugin.